pkg/api/api.go

```diff
@@ -295,16 +295,16 @@ func formatJSONReport(report, prevReport sippyprocessingv1.TestReport, endDay, j
 	data := html.TestReports{
 		Current:      report,
 		Prev:         prevReport,
-		EndDay:       endDay,
+		NumDays:      endDay,
```

Comment:
should the arg name on this function change?  (seems like yes?)

Reply:
yes it should :-)